### DISCUSSION TITLE
Are outgoing spam mail sent?

### DISCUSSION THREAD

**Discussion by kennylajara** [rspamd/rspamd#3773] [Q&A] [answered]
If RSPAMD scan outgoing mails and mark an email as spam, will that mail be sent with some SPAM flag or will that prevent the email to be sent?

**Comment by moisseev** [comment, marked as answer]
It is no different from incoming mail. An email will be sent with spam header if it has triggered `add header` action. A email will be rejected if has triggered ` reject` action.

**Reply by kennylajara** [reply]
Thank you very much for clarifying
